### ISSUE TITLE
Code Bucket Size Suggestion / Ideas

### ISSUE THREAD

**Issue by ChrisCarini** [ChrisCarini/loc-change-count-detector-jetbrains-plugin#163]
This issue is intended to capture different suggestions / ideas for 'buckets of code sizes' (i.e. what constitutes a 'small' change vs a 'large' change; or any terminology one would rather use).

**Comment by ChrisCarini** [comment]
Found on Reddit: 
```
Tiny change - 1 file

Small change - 10 files

Big change - 50 files

Break this up right now change - 100 files
```

**Ref:** https://www.reddit.com/r/programming/comments/wc77ah/comment/iiechs5/

**Comment by ChrisCarini** [comment]
Of course, ChatGPT has some thoughts:

**TL;DR:**
```
Small: <=100
Medium: 100-500
Large: 500-1000
Very Large: 1000+
```

**Screenshot:**
<img width="664" alt="Screenshot 2023-04-16 at 23 48 59" src="https://user-images.githubusercontent.com/6374067/232406167-e7122acd-7061-497b-91bc-15b67e51f984.png">

**Comment by ChrisCarini** [comment]
@Microsoft has a GitHub App to label PRs based on side; sizes found here: https://github.com/microsoft/PullRequestQuantifier/blob/main/prquantifier.yaml

**TL;DR:**

- XS - 50
- S - 100
- M - 200
- L - 400
- XL - 1000

**Comment by ChrisCarini** [comment]
From https://github.com/noqcks/pull-request-size#sizing :

<img width="852" alt="Screenshot 2023-09-21 at 06 14 12" src="https://github.com/ChrisCarini/loc-change-count-detector-jetbrains-plugin/assets/6374067/ca115dc9-716a-4f1a-a65a-a8776c55d8f1">
